### PR TITLE
ScVi Integration

### PR DESCRIPTION
**Pull request by Vaedz7**
Changes
- Added SV folder to Wiki
- Added Documentation for Item Dupe Program
- Add SV to Program List

### CODE REVIEW

File: Wiki/Programs/PokemonSV/SurpriseTrader.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Program Description
+
+This program will surprise trade entire boxes of Pokémon. The main use case is trading away breedjects for new potentially shiny Pokemon!
+
+### Setup of Settings
+
+1. Text Speed: Fast
+2. Cursor Anywhere in starting box
+3. Have entire dex filled up (Preferable but not necessary, if not then there may be errors in recieving Pokemon)
```
Comment (pifopi) [inline Wiki/Programs/PokemonSV/SurpriseTrader.md]: typo recieving

File: Wiki/Programs/PokemonSV/ItemDupe.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Item Dupe
+
+**Related Programs:**
+- **Microcontroller:** [Item Dupe](https://github.com/Vaedz7/Microcontroller/blob/master/Wiki/Programs/PokemonSV/ItemDupe.md) (this program)
```
Comment (pifopi) [inline Wiki/Programs/PokemonSV/ItemDupe.md]: why not move everything to PA ?

File: Wiki/Programs/PokemonSV/TeraAutoHost.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## Program Description
+
+Automatically Host Tera Raids for Discord/Twitch without the use of a Serial Connection!
+
+## Setup of Settings
+### Board Setup
+1. Follow Microcontroller guide by Pokemon Automation [here](https://github.com/PokemonAutomation/Microcontroller "here")
+2. Download program(s) from **THIS REPOSITORY**
+3. Copy and Paste ``TeraAutoHost_Core.c`` to ``CustomProgram_Core.c``
```
Comment (pifopi) [inline Wiki/Programs/PokemonSV/TeraAutoHost.md]: that step is missing for other programs ?

File: Wiki/Programs/PokemonSV/SurpriseTrader.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# Online Surprise Trader
+
+**Related Programs:**
+- **Microcontroller:** [Surprise Trade](https://github.com/PokemonAutomation/Microcontroller/blob/master/Wiki/Programs/PokemonSV/SurpriseTrader.md) (this program)
```
Comment (pifopi) [inline Wiki/Programs/PokemonSV/SurpriseTrader.md:5]: why not move everything to PA ?

File: Wiki/Programs/PokemonSV/TeraAutoHost.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
+# Tera Raid AutoHost in Pokemon Scarlet and Violet
+**CAPTURE CARD REQUIRED**
+
+**Related Programs:**
+- **Microcontroller:** [Tera Raid AutoHost](https://github.com/Vaedz7/Microcontroller/blob/master/Wiki/Programs/PokemonSV/TeraAutoHost.md) (this program)
```
Comment (pifopi) [inline Wiki/Programs/PokemonSV/TeraAutoHost.md]: why not move everything to PA ?

File: Wiki/Programs/PokemonSV/SurpriseTrader.md (hunk 1 above)
Comment (Vaedz7) [reply Wiki/Programs/PokemonSV/SurpriseTrader.md]: Fixed

File: Wiki/Programs/PokemonSV/TeraAutoHost.md (hunk 3 above)
Comment (Vaedz7) [reply Wiki/Programs/PokemonSV/TeraAutoHost.md]: That isn't supposed to be there. Removed.

File: Wiki/Programs/PokemonSV/SurpriseTrader.md (hunk 4 above)
Comment (Vaedz7) [reply Wiki/Programs/PokemonSV/SurpriseTrader.md:5]: Already in PA

File: Wiki/Programs/PokemonSV/ItemDupe.md (hunk 2 above)
Comment (Vaedz7) [reply Wiki/Programs/PokemonSV/ItemDupe.md]: Moved to PA

File: Wiki/Programs/PokemonSV/TeraAutoHost.md (hunk 5 above)
Comment (Vaedz7) [reply Wiki/Programs/PokemonSV/TeraAutoHost.md]: Moved to PA

**Review by pifopi — APPROVED** [review]

**Pull request merged**